kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx

```diff
@@ -92,22 +86,19 @@ const SendMessage: React.FC = () => {
     });
   }, [keyDefaultValue, contentDefaultValue, reset]);
 
-  const onSubmit = async (data: {
-    key: string;
-    content: string;
-    headers: string;
-    partition: number;
-  }) => {
+  console.log(keyDefaultValue);
+
+  const [key, setKey] = useState(keyDefaultValue);
+  const [content, setContent] = useState(contentDefaultValue);
+  const [headers, setHeaders] = useState({});
+  const [partition, setPartition] = useState(0);
+
+  const isValid = key && content && headers;
```

Comment:
Why we trying to reimplement `isValid` method of `react-hook-form`?

Reply:
Fixed